app/routes/admin/reports/system-logs/mail-logs.js

```diff
@@ -0,0 +1,46 @@
+import Ember from 'ember';
+
+const { Route, String } = Ember;
+
+export default Route.extend({
+  titleToken() {
+    return this.i18n.t('Mail Logs');
+  },
+
+  model() {
+    return [
+      {
+        recipient : 'sample@gmail.com',
+        time      : new Date(),
+        action    : 'Session Accept or Reject',
+        subject   : 'Session test proposal has been accepted',
+        message   : String.htmlSafe('Hi y@z.com,<br/>The session <strong>test proposal </strong> has been <strong>accepted</strong> by the organizer. <br/> Visit this link to view the session: http://localhost:5000/events/6/sessions/14/')
```

Comment:
@geekyd I had talked to some orga people, there is a discussion going on regarding this in the orga server (gitter). This won't stay like this is in the next gen,  @niranjan94  Please let me know what to do for now?